parser/parser.go

```diff
@@ -26,9 +26,8 @@
 
 package parser
 
-import __yyfmt__ "fmt"
-
 import (
+	__yyfmt__ "fmt"
```

Comment:
These and the few other unrelated changes were done by my IDE (GoLand) automatically during a merge from upstream but they seem legit to me, so I checked them in.

Reply:
There are triggering the CI failure:

```
[2022-07-24T23:54:47.514Z]  
[2022-07-24T23:54:47.514Z] +import __yyfmt__ "fmt"
[2022-07-24T23:54:47.514Z] +
[2022-07-24T23:54:47.514Z]  import (
[2022-07-24T23:54:47.514Z] -	__yyfmt__ "fmt"
[2022-07-24T23:54:47.514Z]  	"strings"
```

Because it's generated code (from parser.y), it will be reverted by the next person :( So I think it is best to revert.

Reply:
Reverted these changes. It seems I had a pre-commit hook doing import optimisation.